www/src/templates/component-page-template.tsx

```diff
@@ -139,7 +142,10 @@ export default function PageTemplate({
         )}
         <div className="d-flex justify-content-between align-items-start">
           <h1 className="mb-4">{mdx.frontmatter.title}</h1>
-          <LeaveFeedback />
+          <Stack direction="horizontal" gap={3}>
```

Comment:
I decided not to use ButtonGroup, since the buttons are not displayed very well visually